packages/micro.rb

```diff
@@ -3,26 +3,26 @@
 class Micro < Package
   description 'A modern and intuitive terminal-based text editor'
   homepage 'https://micro-editor.github.io/'
-  version '1.4.1'
+  version '2.0.13'
   license 'MIT'
   compatibility 'all'
   binary_compression 'tar.xz'
 
   case ARCH
   when 'aarch64', 'armv7l'
-    source_url 'https://github.com/zyedidia/micro/releases/download/v1.4.1/micro-1.4.1-linux-arm.tar.gz'
-    source_sha256 '3d7a2d1df344fbf0d3bd76f6a6748e490d80c8dfd5f9e8539b25997d459c5411'
+    source_url 'https://github.com/zyedidia/micro/releases/download/v2.0.13/micro-2.0.13-linux-arm.tar.gz'
+    source_sha256 'adb9cf644354a5c85819db40e1a427f0f4951b172597bbcd3ef94ecc4a8c4b75'
   when 'i686'
-    source_url 'https://github.com/zyedidia/micro/releases/download/v1.4.1/micro-1.4.1-linux32.tar.gz'
+    source_url 'https://github.com/zyedidia/micro/releases/download/v2.0.13/micro-2.0.13-linux32.tar.gz'
     source_sha256 'd72d511b4f0fec5bb614b38607e68c5f0918f836d0f60e87c48f0b18e2eab916'
   when 'x86_64'
-    source_url 'https://github.com/zyedidia/micro/releases/download/v1.4.1/micro-1.4.1-linux64.tar.gz'
+    source_url 'https://github.com/zyedidia/micro/releases/download/v2.0.13/micro-2.0.13-linux64.tar.gz'
     source_sha256 'e7d4c9427f9fdfed78e69d42cf518e93ae15fc8f70b7f0f87d292ed81206e900'
   end
 
   binary_sha256({
-    aarch64: '305c70b8ae3ff6e8e80baf8d9af09db29b11fd3e40d381c787ee1f88406cc166',
-     armv7l: '305c70b8ae3ff6e8e80baf8d9af09db29b11fd3e40d381c787ee1f88406cc166',
+    aarch64: 'f242d1b9138c90ae4f9f26b999ecef2e32875fc97d4eb19a8921a52adeb1896b',
+     armv7l: 'f242d1b9138c90ae4f9f26b999ecef2e32875fc97d4eb19a8921a52adeb1896b',
        i686: 'f92d08c76dc125b24674754b3a29ae94bccbfe2bcfca68ae21384e2215054a74',
      x86_64: '64028544e15314a6a576521a7d21e156b9e82eda91fda40033ec19e2c75d1881'
```

Comment:
The i686 and x86_64 sha256 hashes should change too.

Reply:
You can try using Docker for the other architectures if your system is compatible.  See https://github.com/uberhacker/crew-docker.

Reply:
I have a question if this ` binary_sha256` block still needed here?
I see this error during test installation.
![image](https://github.com/user-attachments/assets/290e075c-db1d-4afe-adff-152a851241de)
And it works fine if I remove it.
![image](https://github.com/user-attachments/assets/5a12858e-0dec-4245-aa1c-fb6125a903a0)

Since the binary is already pre-compiled from the source, do we still need to repackage it internally?

Reply:
The binaries are for the uploaded binaries... You don't have access to upload the binaries, so the hashes don't connect to a binary that crew can download.

Just replace your binary section:

```
binary_sha256({
    aarch64: 'f242d1b9138c90ae4f9f26b999ecef2e32875fc97d4eb19a8921a52adeb1896b',
     armv7l: 'f242d1b9138c90ae4f9f26b999ecef2e32875fc97d4eb19a8921a52adeb1896b',
       i686: 'f92d08c76dc125b24674754b3a29ae94bccbfe2bcfca68ae21384e2215054a74',
     x86_64: '64028544e15314a6a576521a7d21e156b9e82eda91fda40033ec19e2c75d1881'
  })
```

with this:
```
no_compile_needed
```

Then you should be ok since it doesn't make sense to generate binaries for this, as the source is already pre-compiled.